src/util/desktop-notifier.js

```diff
@@ -1,22 +1,35 @@
 /* @flow */
-import {NotificationCenter} from 'node-notifier';
+import defaultNotifier from 'node-notifier';
 
-const defaultNotifier = new NotificationCenter({
-  withFallback: true,
-});
+import {createLogger} from './logger';
+
+const log = createLogger(__filename);
 
 type desktopNotificationsParams = {
   title: string,
   message: string,
   icon?: string,
-  notifier?: typeof NotificationCenter,
+};
+
+type desktopNotificationsOpts = {
+  notifier?: typeof defaultNotifier,
+  logger?: typeof log,
 };
 
 export function desktopNotifications(
   {
-    title, message, icon, notifier = defaultNotifier,
-  }: desktopNotificationsParams
+    title, message, icon,
+  }: desktopNotificationsParams,
+  {
+    notifier = defaultNotifier,
+    logger = log,
```

Comment:
@kumar303 This solution to spy the logger seems reasonable to me, how it looks to you?